discovery-provider/src/queries/queries.py

```diff
@@ -604,13 +605,13 @@ def get_ursm_content_nodes():
 
 
 # Get this discovery provider's ipfs peer info
-@bp.route("/ipfs_peer_info", methods=("GET",))
-def get_ipfs_peer_info_route():
-    try:
-        ipfs_peer_info = get_ipfs_peer_info()
-        return api_helpers.success_response(ipfs_peer_info)
-    except exceptions.ArgumentError as e:
-        return api_helpers.error_response(str(e), 400)
+# @bp.route("/ipfs_peer_info", methods=("GET",))
```

Comment:
we should just remove this code probs. i can do as a sep PR if you want to keep this clean